.idea/gradle.xml

```diff
@@ -7,7 +7,7 @@
         <option name="testRunner" value="GRADLE" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="Embedded JDK" />
+        <option name="gradleJvm" value="11" />
```

Comment:
Was this a deliberate switch?

Reply:
Nope, not really. I guess Android Studio did that change automatically. May be it's even better to add the  .idea folder to .gitignore?

Reply:
That's probably a good idea, I've kept it around in the hope it would speed fresh setup, I question how much it's really helped in that sense.